GiniVision/Classes/Core/ContainerNavigationController.swift

```diff
@@ -8,10 +8,15 @@
 
 import UIKit
 
+/**
+ Container that wraps a UINavigationController in order to handle rotation.
+ Parent coordinator should be always `nil` excepts when there is no possibility
```

Comment:
**The** parent coordinator should be always `nil` except<strike>s</strike> when there is no possibility